### ISSUE TITLE
Allign Edit VM component look with the VM Detail

### ISSUE THREAD

**Issue by mareklibra** [oVirt/ovirt-web-ui#200]
Follow-up for https://github.com/oVirt/ovirt-web-ui/pull/197
See https://github.com/oVirt/ovirt-web-ui/pull/197#pullrequestreview-40267230

Required for: https://github.com/oVirt/ovirt-web-ui/issues/156

----------
Edit VM page needs to be rearranged to:

  - Title: [icon] [VM name] - [Edit]
  - order and design of fields similar to VM Detail

**Comment by mareklibra** [comment]
Blocked by https://github.com/oVirt/ovirt-web-ui/pull/197

**Comment by mareklibra** [comment]
Duplicate of https://github.com/oVirt/ovirt-web-ui/issues/196
